src/pages/debrief/cat-be/debrief.cat-be.selector.ts

```diff
@@ -116,14 +116,14 @@ export const getVehicleCheckDrivingFaults =
 
     const showMeDFFault = vehicleChecks.showMeQuestions.filter(fault => fault.outcome === CompetencyOutcome.DF);
     const tellMeDFFault = vehicleChecks.tellMeQuestions.filter(fault => fault.outcome === CompetencyOutcome.DF);
-    if (showMeDFFault.length > 0
-      || tellMeDFFault.length > 0) {
+    const totalFaults = showMeDFFault.length + tellMeDFFault.length;
+    if (totalFaults > 0) {
       const competency: CommentedCompetency & MultiFaultAssignableCompetency = {
         comment: vehicleChecks.showMeTellMeComments || '',
         competencyIdentifier: CommentSource.VEHICLE_CHECKS,
         competencyDisplayName: CompetencyDisplayName.SHOW_ME_TELL_ME,
         source: CommentSource.VEHICLE_CHECKS,
-        faultCount: showMeDFFault.length + tellMeDFFault.length,
+        faultCount: totalFaults === 5 ? 4 : totalFaults,
```

Comment:
We already have `getVehicleChecksFaultCountCatBE` in the FaultCountProvider so we should look to refactor this at some point